mpas_analysis/shared/interpolation/remapper.py

```diff
@@ -267,6 +277,11 @@ def remap_file(self, inFileName, outFileName,
         if variableList is not None:
             args.extend(['-v', ','.join(variableList)])
 
+        # make sure any output is flushed before we add output from the
+        # subprocess
+        sys.stdout.flush()
+        sys.stderr.flush()
```

Comment:
These and the similar commands above make sure that output in the log file shows up in the right order.  Without them, output from calls to `ESMF_RegridWeightGen` and `ncremap` were not necessarily written to the log file in the order they occurred.